languages/python/exercises/concept/elyses-enchantments/.docs/introduction.md

```diff
@@ -0,0 +1,242 @@
+
+## Construction
+
+A list can be declared as a _literal_ with square `[]` brackets and commas between elements:
+
+```python
+>>> no_elements = []
+[]
+
+>>> one_element = ["Guava"]
+["Guava"]
+
+>>> elements_separated_with_commas = ["Parrot", "Bird", 334782]
+["Parrot", "Bird", 334782]
+```
+
+For readability, line breaks can be used when there are many elements or nested data structures within a list:
+
+```python
+>>> lots_of_entries =[
+                        "Rose",
+                        "Sunflower",
+                        "Poppy",
+                        "Pansy",
+                        "Tulip",
+                        "Fuchsia",
+                        "Cyclamen",
+                        "Lavender",
+                        "Daisy",
+                        "Jasmine",
+                        "Hydrangea",
+                        "Hyacinth",
+                        "Peony",
+                        "Dahlia",
+                        "Dandelion",
+                        "Tuberose",
+                        "Ranunculus"
+                      ]
+['Rose', 'Sunflower', 'Poppy', 'Pansy', 'Tulip', 'Fuchsia', 'Cyclamen', 'Lavender', 'Daisy', 'Jasmine', 'Hydrangea', 'Hyacinth', 'Peony', 'Dahlia', 'Dandelion', 'Tuberose', 'Ranunculus']
+
+>>> nested_data_structures = [
+                                 {"fish": "gold", "monkey": "brown", "parrot" : "grey"},
+                                 ("fish", "mammal", "bird"),
+                                 ['water', 'jungle', 'sky']
+                             ]
+[{"fish": "gold", "monkey": "brown", "parrot" : "grey"}, ("fish", "mammal", "bird"), ['water', 'jungle', 'sky']]
+```
+
+The `list()` constructor can be used empty or with an _iterable_ as an argument. Elements in the iterable are cycled through by the constructor and added to the list in order:
+
+```python
+>>> no_elements = list()
+[]
+
+#the tuple is unpacked and each element is added
+>>> multiple_elements_tuple = list(("Parrot", "Bird", 334782))
+["Parrot", "Bird", 334782]
+
+#the set is unpacked and each element is added
+>>> multiple_elements_set = list({2, 3, 5, 7, 11})
+[2,3,5,7,11]
+```
+
+Results when using a list constructor with a string or a dict may be surprising:
+
+````python
+
+#string elements (Unicode code points) are iterated through and added *individually*
+>>> multiple_elements_string = list("Timbuktu")
+['T', 'i', 'm', 'b', 'u', 'k', 't', 'u']
+
+
+>>> multiple_code_points_string = list('अभ्यास')
+['अ', 'भ', '्', 'य', 'ा', 'स']
+
+"""
+The iteration default for dictionaries is over the keys.
+"""
+source_data = {"fish": "gold", "monkey": "brown"}
+>>> multiple_elements_dict_1 = list(source_data)
+['fish', 'monkey']
+
+Because the constructor will only take _iterables_ (or nothing) as arguments, objects that are _not_ iterable will throw a type error. Consequently, it is much easier to create a one-item list via the literal method.
```

Comment:
```suggestion
Because the `list` constructor will only take _iterables_ (or nothing) as arguments, objects that are _not_ iterable will throw a type error. Consequently, it is much easier to create a one-item list via the literal method.
```